toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -594,4 +594,4 @@ zstd-1.5.5-1.cm2.aarch64.rpm
 zstd-debuginfo-1.5.5-1.cm2.aarch64.rpm
 zstd-devel-1.5.5-1.cm2.aarch64.rpm
 zstd-doc-1.5.5-1.cm2.aarch64.rpm
-zstd-libs-1.5.5-1.cm2.aarch64.rpm
\ No newline at end of file
+zstd-libs-1.5.5-1.cm2.aarch64.rpm
```

Comment:
just curious what this zstd-libs diff is - missing EOF or invisible character?

Reply:
yeah this is yet other text file without a EOF newline; since all unix editors add a EOF newline because all unix text files are required to have one, it's actually quite hard for me to edit a text file and *not* have the missing EOF newline automatically fixed (i.e. I didn't add this EOF newline manually; simply editing the release number earlier in the file and then saving it performed the EOF newline correction automatically)

Reply:
it's easier to see with git cli:

```
$ git show toolkit/resources/manifests/package/toolchain_aarch64.txt
commit 2f6893a608a1b2bcce88cc732ab83e803498c4fd (HEAD -> ddstreet/systemd-254, origin/ddstreet/systemd-254)
Author: Dan Streetman <ddstreet@ieee.org>
Date:   Tue Nov 14 18:35:30 2023 -0500

    update to systemd 254

diff --git a/toolkit/resources/manifests/package/toolchain_aarch64.txt b/toolkit/resources/manifests/package/toolchain_aarch64.txt
index e871137fd..81650c388 100644
--- a/toolkit/resources/manifests/package/toolchain_aarch64.txt
+++ b/toolkit/resources/manifests/package/toolchain_aarch64.txt
@@ -555,10 +555,10 @@ sqlite-devel-3.44.0-1.cm2.aarch64.rpm
 sqlite-libs-3.44.0-1.cm2.aarch64.rpm
 swig-4.0.2-3.cm2.aarch64.rpm
 swig-debuginfo-4.0.2-3.cm2.aarch64.rpm
-systemd-bootstrap-250.3-13.cm2.aarch64.rpm
-systemd-bootstrap-debuginfo-250.3-13.cm2.aarch64.rpm
-systemd-bootstrap-devel-250.3-13.cm2.aarch64.rpm
-systemd-bootstrap-rpm-macros-250.3-13.cm2.noarch.rpm
+systemd-bootstrap-250.3-14.cm2.aarch64.rpm
+systemd-bootstrap-debuginfo-250.3-14.cm2.aarch64.rpm
+systemd-bootstrap-devel-250.3-14.cm2.aarch64.rpm
+systemd-bootstrap-rpm-macros-250.3-14.cm2.noarch.rpm
 tar-1.34-2.cm2.aarch64.rpm
 tar-debuginfo-1.34-2.cm2.aarch64.rpm
 tdnf-3.5.2-2.cm2.aarch64.rpm
@@ -594,4 +594,4 @@ zstd-1.5.5-1.cm2.aarch64.rpm
 zstd-debuginfo-1.5.5-1.cm2.aarch64.rpm
 zstd-devel-1.5.5-1.cm2.aarch64.rpm
 zstd-doc-1.5.5-1.cm2.aarch64.rpm
-zstd-libs-1.5.5-1.cm2.aarch64.rpm
\ No newline at end of file
+zstd-libs-1.5.5-1.cm2.aarch64.rpm

```